OpenRA.Mods.Common/Widgets/Logic/Ingame/OrderButtonsChromeLogic.cs

```diff
@@ -47,20 +49,25 @@ public PowerdownOrderButtonLogic(Widget widget, World world)
 
 	public class BeaconOrderButtonLogic : ChromeLogic
 	{
+		static string beaconcursor;
+		public static Action<BeaconOrderGenerator> AssignBeaconCursor = (orderGenerator) => orderGenerator.BeaconCursor = beaconcursor;
```

Comment:
You can remove parentheses, they're not necessary, when you have only one parameter.